charts/agent/values.yaml

```diff
@@ -152,9 +150,7 @@ cluster-events:
 
   image:
     repository: observeinc/observe-agent
-    # Overrides the image tag whose default is the chart appVersion.
-    # The service's name will be applied to the end of this value.
-    tag: "1.3.0"
+    tag: latest
```

Comment:
Hum... Not pinning the image is likely too dangerous to be used as default. As user could see their agent "secretively" updating. Why do we need this as default?